ls1intum/Artemis
# `General`: Add course title to enrollment confirmation

### Checklist
#### General



#### Client



### Motivation and Context
This PR adds the course Title to the modal title of the Course Enrollment confirmation popup as requested by Stephan.

### Steps for Testing

Prerequisites:
- 1 Student 
- 1 Course with active Course Enrollment and Confirmation Message 

1. Log in to Artemis
2. Navigate to Course Enrollment
3. Enroll for this course
4. Verify that there is the course title within the Modal title

### Testserver States
> [!NOTE]
> These badges show the state of the test servers.
> Green = Currently available, Red = Currently locked

![](https://byob.yarr.is/ls1intum/Artemis/artemis-test1)
![](https://byob.yarr.is/ls1intum/Artemis/artemis-test2)
![](https://byob.yarr.is/ls1intum/Artemis/artemis-test3)
![](https://byob.yarr.is/ls1intum/Artemis/artemis-test4)
![](https://byob.yarr.is/ls1intum/Artemis/artemis-test5)
![](https://byob.yarr.is/ls1intum/Artemis/artemis-test6)
![](https://byob.yarr.is/ls1intum/Artemis/artemis-test9)

### Screenshots
![image](https://github.com/ls1intum/Artemis/assets/78978542/e044d063-c815-4238-a8bd-addcfbc2fde8)
